lib/Doctrine/ORM/UnitOfWork.php

```diff
@@ -426,7 +428,11 @@ public function commit()
 
         $this->dispatchPostFlushEvent();
 
-        // Clean up
+        $this->postCommitCleanup();
+    }
+
+    private function postCommitCleanup()
```

Comment:
```suggestion
    private function postCommitCleanup() : void
```

Reply:
I did not add `void` because other methods without return type-hints.

Reply:
`master` has enforced CS fixes for diffs anyway, so if this isn't applied you'll get a red build anyway

Reply:
Fixed